tuna/go_fish.py

```diff
@@ -71,11 +71,11 @@ def parse_args():
                       help=
                       'Session ID to be used as tuning tracker. ' \
                       'Allows to correlate DB results to tuning sessions')
-  parser.add_argument('--local_machine',
-                      dest='local_machine',
+  parser.add_argument('--remote_machine',
```

Comment:
where is this used?

Reply:
--local_machine was used everywhere so the inverse --romote_machine is used nowhere. But this allows use of the older functionality if desired.

Reply:
i see ok, if you think its worth keeping then we should keep it. Please document this somewhere